bloom-housing/bloom
# 1975/one month rent

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/1975)


## Description
Adds the deposit helper text field in the following places/ways:
1) in partners listing form there should be a field for deposit helper text next to the "costs not included" field
2) in partners listing detail page there should be a field for deposit helper text to the "costs not included" field
3) on the public listing, if there was text in the deposit helper text for that listing then that text should display under the min-max deposit range

## How Can This Be Tested/Reviewed?
Create or edit a listing
add or remove text from the new deposit helper text field
verify that change is reflected in the partner side detail page for that listing
verify that change is reflected in the public side detail page for that listing

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
